spec/common/local-biblio.js

```diff
@@ -1,4 +1,14 @@
 var localBibliography = {
+  CBD: {
```

Comment:
Detail: CBD is in Specref

Reply:
Must be new. We’ll queue it up to remove in another round.